assets/pages/new-contributions-landing/contributionsLandingActions.js

```diff
@@ -93,9 +93,11 @@ const setGuestAccountCreationToken = (guestAccountCreationToken: string): Action
 const setThankYouPageStage = (thankYouPageStage: ThankYouPageStage): Action =>
   ({ type: 'SET_THANK_YOU_PAGE_STAGE', thankYouPageStage });
 
-const isPaymentReady = (paymentReady: boolean, paymentHandlers: ?{ [PaymentMethod]: PaymentHandler }): Action =>
-  ({ type: 'UPDATE_PAYMENT_READY', paymentReady, paymentHandlers: paymentHandlers || null });
-
+const isPaymentReady = (
+  paymentReady: boolean,
+  paymentHandlers: ?{ [Contrib]: { [PaymentMethod]: PaymentHandler }},
+): Action =>
+  ({ type: 'UPDATE_PAYMENT_READY', paymentReady, thirdPartyPaymentLibraries: paymentHandlers || null });
```

Comment:
I'm surprised you can override `thirdPartyPaymentLibraries`, there is no guarantee that `paymentHandler` will have the correct shape with the required values, is there?

Reply:
You mean because are no types here?

```javascript
function initialiseStripeCheckout(onPaymentAuthorisation, contributionType, currencyId, isTestUser, dispatch) 
```

Could be a weakness in Flow. Since I think untyped arguments get an `any` type, and the implied `contributionType: any` is ultimately being used as a key required to be `Contrib`, I would've thought it would complain